bbb-convert-stage-5.sh

```diff
@@ -113,26 +113,18 @@ build_grub_efi() {
   fi
 
   mkdir -p $grub_arm
```

Comment:
Rename $grub_arm to $grub_build - let it be "local grub_build=$output_dir/grub/build"